debian/changelog

```diff
@@ -0,0 +1,23 @@
+gstd (0.11.3-1) unstable; urgency=medium
```

Comment:
You can automate this to avoid further errors caused by developers. You can have a look at: https://github.com/lleon95/debian-example-ci